src/utils/update/managing.officer.mapper.ts

```diff
@@ -75,20 +67,19 @@ export const splitNames = (officerName: string) => {
   }
 };
 
-export const getFormerNames = (formerNames?: FormerNameResource[]) => {
-  let allFormerNames;
-
+export const getFormerNames = (formerNames?: FormerNameResource[]): string => {
+  let allFormerNames = "";
   if (formerNames !== undefined) {
     for (let loop = 0; loop < formerNames.length; loop++) {
       const forenames = formerNames[loop].forenames;
       const surname = formerNames[loop].surname;
-      const holder = forenames?.concat(" " + surname);
+      const holder = forenames ? forenames.concat(" " + surname) : surname;
```

Comment:
Is it possible to only have surname? 
const holder = forenames?.concat(" " + surname); might be a safer bet

Reply:
If fornames is undefined then holder was undefined. Seems safer to just use surname.